base/linalg/symmetric.jl

```diff
@@ -300,7 +300,6 @@ A_mul_B!(C::StridedMatrix{T}, A::StridedMatrix{T}, B::Hermitian{T,<:StridedMatri
     BLAS.hemm!('R', B.uplo, one(T), B.data, A, zero(T), C)
 
 *(A::HermOrSym, B::HermOrSym) = full(A)*full(B)
```

Comment:
That's a good idea actually, that’s also what the timings suggest above. (e.g. when choosing between `full`ing the first or second)